index.html

```diff
@@ -0,0 +1,89 @@
+					<img src="images/ProjectPlaceholder.jpg" alt="Placeholder project" tabindex="4" />
+					<h2 tabindex="4">Placeholder Project</h2>
+					<p tabindex="4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+				</article>
+				<article class="box">
+					<img src="images/ProjectPlaceholder.jpg" alt="Placeholder project" tabindex="4" />
+					<h2 tabindex="4">Placeholder Project</h2>
+					<p tabindex="4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+				</article>
+				<article class="box">
+					<img src="images/ProjectPlaceholder.jpg" alt="Placeholder project" tabindex="4" />
+					<h2 tabindex="4">Placeholder Project</h2>
+					<p tabindex="4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+				</article>
+				<article class="box">
+					<img src="images/ProjectPlaceholder.jpg" alt="Placeholder project" tabindex="4" />
+					<h2 tabindex="4">Placeholder Project</h2>
+					<p tabindex="4">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+				</article>
+			</div>
+		</main>
+		<footer>
+			<a class="anchor-target" id="contact"></a>
+			<h1 tabindex="5">contact</h1>
+			<div>
+				<div>
+					<a href="mailto:Matthew.wr89@gmail.com" tabindex="5">email<span>: Matthew.wr89@gmail.com</span></a>
+					<a href="https://github.com/MattW-R" tabindex="5">github<span>: MattW-R</span></a>
+					<a href="" tabindex="5">LinkedIn<span>: Matthew W-R</span></a>
```

Comment:
awaiting url

Reply:
Need to re-setup LinkedIn, its currently dormant. Was waiting till you talk about it later.

Reply:
Sounds good